third_party/move/move-model/bytecode/src/usage_analysis.rs

```diff
@@ -28,7 +29,7 @@ pub fn get_memory_usage<'env>(target: &FunctionTarget<'env>) -> &'env UsageState
 }
 
 /// A summary of the memory accessed / modified per function, both directly and transitively.
-#[derive(Default, Clone)]
+#[derive(AbstractDomain, Default, Clone)]
```

Comment:
Don't you need to add #[join] to the various fields?

Similarly for below.

Reply:
done